src/main/resources/static/views/login/login.html

```diff
@@ -1,89 +1,107 @@
-<div class="container">
-    <div class="row">
-        <div class="col-md-6 col-md-offset-3">
-            <div class="panel panel-login">
-                <div class="panel-heading">
-                    <div class="row">
-                        <div class="col-xs-6">
-                            <a href="#" class="active" id="login-form-link">Login</a>
-                        </div>
-                        <div class="col-xs-6">
-                            <a href="#" id="register-form-link">Register</a>
-                        </div>
-                    </div>
-                    <hr>
-                </div>
-                <div class="panel-body">
-                    <div class="row">
-                        <div class="col-lg-12">
-                            <form id="login-form" action="#" method="post" role="form" style="display: block;">
-                                <div class="form-group">
-                                    <input type="text" name="username" id="username" tabindex="1" class="form-control" placeholder="Email" value="">
-                                </div>
-                                <div class="form-group">
-                                    <input type="password" name="password" id="password" tabindex="2" class="form-control" placeholder="Password">
+<body>
+<div class="container-fluid">
+    <div class="row content">
+        <div class="col-sm-3 sidenav hidden-xs">

```

Comment:
I think this side nav should be in the index.html because i see it repeated a lot. We can add an ng-if to show it when the user is authenticated. We don't have to change it now, maybe when we implement the authentication.

Reply:
I absolutely agree, we definitely should consolidate that into the index.html. We can toggle the active using JS.

Reply:
Once we do this we can then call `$('.sidenav').css("height",$(document).height() + "px");` only once as opposed to in all controllers.